ui/media/js/parameters.js

```diff
@@ -420,6 +420,8 @@ async function getSystemInfo() {
         let force = false
         if (res['enforce_output_dir'] !== undefined) {
             force = res['enforce_output_dir']
+            saveToDiskField.checked = true
```

Comment:
I think this should be set to true only if `res['enforce_output_dir']` is true (i.e. the `force` variable is true).

And if `force` is false, we shouldn't modify this field (and let the auto-save logic do its thing).

Basically `res['enforce_output_dir']` can be `false` for other reasons (for e.g. if the user never set `force_save_path` in `config.json`), in which case we shouldn't modify this toggle.

Same comment for the next line.

Reply:
ok, I'll check if force == true